apps/core/app/(default)/(faceted)/brand/[slug]/static/page.tsx

```diff
@@ -14,3 +15,4 @@ export async function generateStaticParams() {
 
 export const dynamic = 'force-static';
 export const revalidate = 600;
+export const runtime = envRuntime;
```

Comment:
I think we want to force this to `nodejs` so it can static render right? Thought the idea is that you configure which runtime you want for the dynamic pages, without breaking the static ones.